btcec/schnorr/musig2/nonces.go

```diff
@@ -368,17 +371,18 @@ func AggregateNonces(pubNonces [][PubNonceSize]byte) ([PubNonceSize]byte, error)
 	if err != nil {
 		return finalNonce, err
 	}
+
 	combinedNonce2, err := combineNonces(func(n [PubNonceSize]byte) []byte {
 		return n[btcec.PubKeyBytesLenCompressed:]
 	})
 	if err != nil {
 		return finalNonce, err
 	}
 
-	copy(finalNonce[:], combinedNonce1.SerializeCompressed())
+	copy(finalNonce[:], btcec.JacobianToByteSlice(combinedNonce1))
 	copy(
 		finalNonce[btcec.PubKeyBytesLenCompressed:],
-		combinedNonce2.SerializeCompressed(),
+		btcec.JacobianToByteSlice(combinedNonce2),
```

Comment:
Looks like a test case expects an actual point, but then we serialize our "all zero" infinity point here (see CI).

Reply:
The test case should actually expect the infinity point (see https://github.com/jonasnick/bips/blob/8ddf79646ce9460896c9f5a74c3583ba6e405cb4/bip-musig2/reference.py#L504).

I broke that in the last force push. Fixed it in my last diff.